AutomationHelpers/src/UserCodeCollections/PopupWatcherLibrary.cs

```diff
@@ -39,6 +39,9 @@ public static class PopupWatcherLibrary
         [UserCodeMethod]
         public static PopupWatcher StartPopupWatcher(RepoItemInfo findElement, RepoItemInfo clickElement)
         {
+            CheckArgumentNotNull(findElement, "findElement");

```

Comment:
How can we require >C#6.0 in the long run? C#5.0 is included in .NET 4.5, but for C#6.0 and above you need the MSBuild Tools to be separately installed.

Reply:
Even if Studio supports C#6.0, you might not have a C#6 compiler installed. So we would need to also cope with that... But not a discussion for this thread